src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/ReviewUnusedParameters.cs

```diff
@@ -250,6 +256,13 @@ bool IsSingleStatementBody(IBlockOperation body)
                     }
                 }
 
+                // Do not raise warning for unused 'this' parameter of an extension method.
+                if (_method.IsExtensionMethod)
```

Comment:
Can you please add a unit test for this case in analyzer test file `ReviewUnusedParametersTests.cs`?